cvat-core/tests/api/annotations.js

```diff
@@ -351,6 +373,68 @@ describe('Feature: put annotations', () => {
             zOrder: 0,
         })).toThrow(window.cvat.exceptions.ArgumentError);
     });
+
+    test('put a skeleton shape to a job', async() => {
+        const job = (await window.cvat.jobs.get({ jobID: 40 }))[0];
+        const label = job.labels[0];
+        await job.annotations.clear(true);
+        await job.annotations.clear();
```

Comment:
Why double clear?

Reply:
Yes, first ``clear`` with ``true`` argument fetches annotations from the server. 
It is necessary, otherwise we are not able to work with collection. 

The second ``clear`` is just to make the check independent on current collection state (there may be some objects, there may be more of them in the future; Below we are checking annotations length, which should be equal 1, because we added only 1 object)